client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import SingleAppContainer from './SingleAppContainer.js';
+import {topFree, topGrossing, topPaid, trending} from './mockData.js';
+import './TopChartsStyle.css';
+import '../styles.css';
+ 
+class TopCharts extends Component {
+  render() {
+    const makeColumn = item => (
+        <div key={item.appID}>
+            <SingleAppContainer
+            listNum={item.listNum}
+            url={item.url}
+            title={item.title}
+            category={item.category}
+            rating={item.rating}
+            />
+        </div>
+    );
+
+    const charts = [topFree, topGrossing, trending, topPaid];
+
+    return (
+      <div className="section">
+        <div class="section-header">
+          <h1>Top Charts</h1>
```

Comment:
This could be done in a later PR, but I would be vary of hard coding any mock data. Data such as cluster title and subtitles should be coming from the backend.